### PR TITLE
docs: Update docs for next SDK release

### PR DESCRIPTION
**Pull request by yagopv**
We are updating the docs with some breaking changes we are including in the next release of the Safe{Core} SDK

### CODE REVIEW

File: pages/sdk-protocol-kit/reference.md [hunk 1]
```diff
@@ -225,7 +225,7 @@ Returns a new instance of the Protocol Kit connected to a new Safe or a new Sign
 Connection of a deployed Safe using the `safeAddress` property:
 
 ```typescript
-const safeSdk = await safeSdk.connect({ ethAdapter, safeAddress })
+const protocolKit = await protocolKit.connect({ ethAdapter, safeAddress })
```
Comment (dasanra) [inline pages/sdk-protocol-kit/reference.md:228]: I think we have a mistake here. Should be:

```suggestion
const protocolKit = await Safe.connect({ ethAdapter, safeAddress })
```

File: pages/sdk-protocol-kit/reference.md [hunk 2]
```diff
@@ -238,7 +238,7 @@ const predictedSafe: PredictedSafeProps = {
   safeDeploymentConfig
 }
 
-const safeSdk = await safeSdk.connect({ ethAdapter, predictedSafe })
+const protocolKit = await protocolKit.connect({ ethAdapter, predictedSafe })
```
Comment (dasanra) [inline pages/sdk-protocol-kit/reference.md:241]: I think we have a mistake here. Should be: 
```suggestion
const protocolKit = await Safe.connect({ ethAdapter, predictedSafe })
```

File: pages/sdk-api-kit/reference.md [hunk 3]
```diff
@@ -236,21 +236,54 @@ const allTxs: SafeMultisigTransactionListResponse = await safeApiKit.getAllTrans
...
+```typescript
+const message: SafeMessage = await apiKit.getMessage(safeMessageHash)
+```
+
+### `getMessages`
+
+Get the list of messages associated to a Safe account
+
+```typescript
+const messagesResponse: SafeMessageListResponse = await apiKit.getMessages(safeAddress, {  ordering, limit, offset })
```
Comment (dasanra) [inline pages/sdk-api-kit/reference.md:271]:
```suggestion
const messagesResponse: SafeMessageListResponse = await apiKit.getMessages(safeAddress, { ordering, limit, offset })
```

File: pages/sdk-protocol-kit/reference.md (hunk 1 above)
Comment (yagopv) [reply pages/sdk-protocol-kit/reference.md:228]: mmm nope, it's ok like that

**Comment by germartinez** [comment]
I'm closing this PR in favor of this other one: https://github.com/safe-global/safe-docs/pull/317
Please read the description for more info.

**Pull request closed without merging**